lib/index.js

```diff
@@ -75,6 +80,16 @@ info: Storage engine shut down.\n";
       process.stderr.write(output);
     }
   });
+
+  self._server = server;
+}
+
+Server.prototype.stop = function(cb) {
+  var self = this;
+  self._server.close(cb);
```

Comment:
it does work, but it's a fair point. will close connections before shutting down server